src/pybind/rbd/rbd.pyx

```diff
@@ -2642,6 +2642,29 @@ cdef class Group(object):
 
     def __exit__(self, type_, value, traceback):
         return False
+    
+    def group_get_id(self):
```

Comment:
Need to rename the actual method as well, not just the call in the test -- please make sure to run the test before re-pushing:
```suggestion
    def id(self):
```

Reply:
sorry, I was careless

Reply:
hi @idryomov Is there any relevant documentation for testing pybind, thanks

Reply:
Probably not -- it should be just `python3 -m pytest ../src/test/pybind/test_rbd.py`.  You may need to set `LD_LIBRARY_PATH` and `PYTHONPATH` environment variables appropriately.

Reply:
... and you can filter out all other tests by appending something like `-k test_group_get_id`.